PilotDataPlatform/bff-web
# PILOT-1556: Add support for project-admin to query project statistics

## Summary

- Allow project-admin role to query project statistics `POST /v1/project/{project_code}/statistics`.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1566

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Login with user that has project admin role 
2. Go to project with project-admin role
3. Project statistics should appear
